src/libs/vue-h-zoom.js

```diff
@@ -1,3 +1,11 @@
+const DEFAULT_BACKGROUND_OPTIONS = {
```

Comment:
please put this default value in the readme

Reply:
updated